docs/cloud/managing-airbyte-cloud.md

```diff
@@ -259,6 +287,23 @@ To display **Connection State**:
 
     The **Connection State** displays. 
 
+## Choose the data residency for a connection
+To choose the data residency for an existing connection: 
+
+1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click **Connections** in the navigation bar and then click the connection that you want to change. 
+
+    The Connection page displays. 
+
+2. Click the **Settings** tab. 
+
+3. Click the **Data residency** dropdown and choose the location for your default data residency.
```

Comment:
Should we add a note here about whether changing a connection's data residency will not affect currently running syncs? @timroes mentioned this when reviewing the UI PR. I don't _think_ currently running syncs will be affected, but I'm not 100% sure. Maybe @davinchia has an idea?

Reply:
Thanks for reminding me Joey - yeah updating the workspace doesn't update existing connections so we should call that out too.

@sophia-wiley one more follow up change!

Reply:
Okay so just to make sure I understand this correctly -- If you choose a different data residency in the connection settings while a sync is running, it will not affect the currently running sync, but it will apply the new data residency to future syncs for that connection?

Reply:
* Data residency changes will only affect the next sync, and not any currently running sync.
* Changing the workspace default residency only affects newly created connections. Already created connections have to be manually edited.

Does ^ make the behaviour clearer?